### ISSUE TITLE
Issue with orientation

### ISSUE THREAD

**Issue by bossdm** [resilient-swarms/argos-sferes#15]
It seems there is some issue with getting the orientation of the robot. 

I noticed that at the Reset of the baseloopfunctions, there was a discrepancy between the angle obtained from the m_vecInitSetup and the angle obtained directly from the robot just after the reset. Actually, if you summed their values you would get 2*Pi, for example, one would have 0.1 and the other would have 2*Pi - 0.1 radians.

Similarly, when running :
> bin/behaviour_evolcvt10D experiments/Gomes_walls_and_robots_std.argos

I can see from print statements the following kind of pattern for some robots in some random seeds:
...
orientation=2*Pi - 0.8
orientation=2*Pi - 0.5
orientation=2*Pi - 0.2 
orientation=2*Pi - 0.1  # starts going back from 2Pi rather than from 0
orientation=2*Pi - 0.4
orientation=2*Pi - 0.7

in other words, the robot is moving at constant rotation velocity but then at 2Pi, the orientation starts going back from 2Pi rather than starting in the same direction from 0.

I have kept the print statements for now so you can also see them.

**Comment by bossdm** [comment]
You can see the behaviour easily when you put lines

>      cController->m_fLeftSpeed = -cController->m_sWheelTurningParams.MaxSpeed; //outf[0];
        cController->m_fRightSpeed = cController->m_sWheelTurningParams.MaxSpeed//outf[1];

on lines 271 of evol_loop_functions.cpp. Turning on visualisation, at least I can see that there is no issue with the actual movement of the robot.

**Comment by bossdm** [comment]
can close this now as the solution was to convert to Euler angles
